README.md

```diff
@@ -1 +1,2 @@
-# yogdaan.github.io
\ No newline at end of file
+# yogdaan.github.io
+This is the website for yogdaan.github.io. <br>we aim to build, promote and use OSS. Anyone is allowed to come up with an idea and everyone just gets along with it and start working.
```

Comment:
can we please remove `<br>` and follow markdown syntax?

Reply:
alright I am good with that